packages/ui/src/components/repo-subheader.tsx

```diff
@@ -16,7 +16,13 @@ export enum RepoTabsKeys {
 
 export const repoTabsKeysArr = Object.values(RepoTabsKeys)
 
-export const RepoSubheader = ({ useTranslationStore }: { useTranslationStore: () => TranslationStore }) => {
+export const RepoSubheader = ({
+  useTranslationStore,
+  showPipelinesTab = true
+}: {
+  useTranslationStore: () => TranslationStore
+  showPipelinesTab?: boolean
```

Comment:
Generally it's a good pattern to default to `false` for optional booleans, that way they can be controlled with existence as a prop, rather than a value.
```suggestion
  hidePipelinesTab = false
}: {
  useTranslationStore: () => TranslationStore
  hidePipelinesTab?: boolean
```

When calling you can just use `<RepoSubheader hidePipelinesTab ... />` instead of `<RepoSubheader showPipelinesTab={false} ... />`

Reply:
Agree. Will make the change.